examples/rbac/client/admin_user_resources.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: feast-admin-sa
+  namespace: feast-dev
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: feast-admin-role
+  namespace: feast-dev
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: feast-admin-rolebinding
+  namespace: feast-dev
+subjects:
+  - kind: ServiceAccount
+    name: feast-admin-sa
+    namespace: feast-dev
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: feast-admin-role
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: client-admin-user
+  namespace: feast-dev
+  labels:
+    app: client-admin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: client-admin
+  template:
+    metadata:
+      labels:
+        app: client-admin
+    spec:
+      serviceAccountName: feast-admin-sa
+      initContainers:
+        - name: init-client-admin
+          image: quay.io/ahameed/feature-server:dev
```

Comment:
Just missed to update the image this will set Feast image -> `feastdev/feature-server`

Reply:
+1

Reply:
@redhatHameed remember to apply this change in the unext PR for RBAC examples

Reply:
Thanks @dmartinol will open the PR separately for examples so this can resolve for now